Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM mhart/alpine-node
 WORKDIR /usr/src/app
 COPY package.json /usr/src/app/
 
+RUN apk --update add git build-base && \
+    apk del build-base && rm -fr /usr/share/ri
```

Comment:
Yes we do that everywhere. [Another example](https://github.com/codeclimate/codeclimate/blob/master/Dockerfile#L11)